package.json

```diff
@@ -23,13 +24,15 @@
     "query-string": "^8.1.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-hook-form": "^7.45.4",
     "react-syntax-highlighter": "^15.5.0",
     "react-transition-group": "^4.4.5",
     "use-clipboard-copy": "^0.2.0",
     "use-query-params": "^2.2.1",
     "viem": "^1.6.0",
     "wagmi": "^1.3.9",
-    "web3modal": "^1.9.12"
+    "web3modal": "^1.9.12",
```

Comment:
I know it's only an update here, but any reason we use `web3modal` over rainbowkit?